packages/@sfcc-core/core/src/index.ts

```diff
@@ -1,18 +1,29 @@
-/*
-    Copyright (c) 2020, salesforce.com, inc.
-    All rights reserved.
-    SPDX-License-Identifier: BSD-3-Clause
-    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
-*/
-/*
-   SFRA Core Code
-    */
-
+import { Logger } from '@sfcc-core/logger';
 export const LOGGER_KEY = Symbol('Logger Service');
 export const API_EXTENSIONS_KEY = Symbol('API Extensions');
 
+export type Extension = any;
+export type Service = any;
+export type ExtensionFactory = () => Extension;
+
+class UnknownServiceError extends Error {
```

Comment:
Define a specific error subclasses the native Error. This is for internal logic (https://github.com/SalesforceCommerceCloud/sfcc-sample-apps/pull/112/files#diff-af688c34e8eb77909dc5001c6a36a19eR114), but could be a type that we eventually expose

Reply:
Yeah. We should definitely bust this out (and use index.ts for exports - as mentioned earlier)